src/collections/_documentation/workflow/integrations/global-integrations.md

```diff
@@ -1082,6 +1082,60 @@ The Split integration is only available for organizations on the Business and En
 Follow the instructions in the link below:  
 <https://help.split.io/hc/en-us/articles/360029879431>
 
+## Deployment
+
+These integrations will assist in automating your release and deployment process, making understanding your errors simpler.
+
+### Vercel
+
+Vercel is an all-in-one platform with Global CDN supporting static & JAMstack deployment and Serverless Functions. Connect your Sentry and Vercel projects to automatically upload source maps and notify Sentry of new releases being deployed.
+
+**Installation**
+
+1. In Sentry, navigate to **Organization Settings** > **Integrations**.
+
+2. Find Vercel in the list of the available Global Integrations list and click **Vercel**.
+
+    [{% asset vercel/vercel_install_sentry.png alt="Sentry modal showing Vercel installation." %}]({% asset vercel/vercel_install_sentry.png @path %})
+
+3. Click the "Vercel Marketplace" button at the top right to install the integration.
+
+    [{% asset vercel/vercel_marketplace.png alt="Sentry modal showing Vercel marketplace button." %}]({% asset vercel/vercel_marketplace.png @path %})
+
+4. Select your account and click "Add".
+
+    [{% asset vercel/vercel_install.png alt="Sentry modal showing Vercel installation." %}]({% asset vercel/vercel_install.png @path %})
+
+    **Note that the installation created an internal integration for the purpose of creating an auth token used in creating releases. If the Vercel internal integration is deleted, your Vercel integration will stop working!**
+
+    [{% asset vercel/vercel_internal_integration.png alt="Sentry modal showing Vercel internal integration." %}]({% asset vercel/vercel_internal_integration.png @path %})
+
+**Project Linking**
+
+1. Select a Sentry project and a Vercel project to link together.
+
+    [{% asset vercel/vercel_link_project.png alt="Sentry modal showing linking Sentry project to Vercel project." %}]({% asset vercel/vercel_link_project.png @path %})
+
+This step will generate environment variables in the selected Vercel project. You can see these in Vercel in **Project Settings** > **General** > **Environment Variables**.
```

Comment:
@ceorourke I think it would be good to show a screenshot of the project environment variables in Vercel as well as as a description here what each environment variable does.

Reply:
sounds good, I wasn't sure about whether or not to add this.